cl/recap/mergers.py

```diff
@@ -459,17 +511,35 @@ def calculate_recap_sequence_numbers(docket_entries):
 
     # Assign sequence numbers
     for prev, de, _ in previous_and_next(docket_entries):
-        if prev is not None and de["date_filed"] == prev["date_filed"]:
+        current_date_filed = de["date_filed"]
+        if isinstance(de["date_filed"], datetime):
+            current_date_filed = convert_to_court_timezone(
+                de["date_filed"], court_id
+            ).date()
+
+        prev_date_filed = None
+        if prev is not None:
+            prev_date_filed = prev["date_filed"]
+        if prev is not None and isinstance(prev["date_filed"], datetime):
+            prev_date_filed = convert_to_court_timezone(
+                prev["date_filed"], court_id
+            ).date()
```

Comment:
You know I hate deep nesting, but I think this is clearer:

```suggestion
        if prev is not None:
            if isinstance(prev["date_filed"], datetime):
                prev_date_filed = convert_to_court_timezone(
                    prev["date_filed"], court_id
                ).date()
            else:
                prev_date_filed = prev["date_filed"]
```

Reply:
I agree, this is clearer, changed!